sendtx_testnet.json

```diff
@@ -2,10 +2,7 @@
   "jsonrpc": "2.0",
   "method": "icx_sendTransaction",
   "params": {
-    "version": "0x3",
```

Comment:
Is there any reason to remove them?

Reply:
Only the required params are left in the json file which is for sendtx through tbears command.

Reply:
This file was deleted since the icx transfer method was changed.